.github/workflows/ci_extended.yml

```diff
@@ -1,7 +1,7 @@
 ---
 name: ci extended
 # yamllint disable-line rule:truthy
-on: push
+on: [push, pull_request]
```

Comment:
I assume you do that just for testing. Please revert before merging

Reply:
yes - as mentioned in the first comment.